src/agoradatatools/etl/transform/biomarkers.py

```diff
@@ -0,0 +1,73 @@
+"""
+This module contains the transformation logic for the biomarkers dataset.
+This is for the Model AD project.
+"""
+
+import pandas as pd
+
+
+def transform_biomarkers(datasets: dict) -> list:
+    """
+    Takes dictionary of dataset DataFrames, extracts the biomarkers
+    DataFrame, and transforms it into a list of dictionaries grouped by
+    'model', 'type', 'ageDeath', 'tissue', and 'units'.
+
+    Args:
+        datasets (dict[str, pd.DataFrame]): dictionary of dataset names mapped to their DataFrame
+
+    Returns:
+        list[dict[str, Any]]: a list of dictionaries containing biomarker data modeled after intended final JSON structure
```

Comment:
We do have some code in place to basically do your transform in the form of a data frame instead of a list, where each row is a grouping and it contains a column with 'nested' data (it's the function `nest_fields` in `utils.py`). See the `genes_biodomains.py` transform for an example of how this is called. So in my head your grouping would be `['model', 'type', 'ageDeath', 'tissue', units']`, and you would want it to nest `['genotype', 'measurement', 'sex']` into a new column called `points`. However, we've only used/tested `nest_fields` with _one_ column as the grouping, never with multiple columns like is needed here, so I'm not sure this will work here. It might be simpler to leave the code as you've written it for now, take another look at `nest_fields` at some point to make it more generic to any number of columns, and edit later, but maybe @JessterB can weigh in.

Reply:
My 2c: I forgot about `nest_fields`... ideally we would use `nest_fields` for this, but if that will add a significant amount of work then we shouldn't tackle it as part of this PR. We can add a ticket to the backlog for extending `nest_fields` and move forward with the existing implementation.

Reply:
I did consider this approach but felt it was extra computational work that could be simplified with keeping the data as a list. However, I do agree that in terms of codebase maintenance it would be best for us to use existing approaches, because I did have to make a few changes to accommodate the new type of transform output. I can definitely update the transform to output a `pd.DataFrame` instead of a `list`. Thoughts?